javalanglib/src/main/scala/java/lang/Character.scala

```diff
@@ -253,28 +303,39 @@ object Character {
   @inline def isSurrogatePair(high: scala.Char, low: scala.Char): scala.Boolean =
     isHighSurrogate(high) && isLowSurrogate(low)
 
-  @inline def toCodePoint(high: scala.Char, low: scala.Char): scala.Int =
+  @inline def toCodePoint(high: scala.Char, low: scala.Char): Int =
     ((high & SurrogateUsefulPartMask) << 10) + (low & SurrogateUsefulPartMask) + 0x10000
 
   // --- End of UTF-16 surrogate pairs handling ---
 
-  def isUnicodeIdentifierStart(c: scala.Char): scala.Boolean =
-    reUnicodeIdentStart.test(c.toString)
-
-  def isUnicodeIdentifierPart(c: scala.Char): scala.Boolean =
-    isUnicodeIdentifierStart(c) || isIdentifierIgnorable(c) ||
-    reUnicodeIdentPartExcl.test(c.toString)
-
-  def isIdentifierIgnorable(c: scala.Char): scala.Boolean =
-    reIdentIgnorable.test(c.toString)
+  def isLowerCase(c: scala.Char): scala.Boolean =
+    isLowerCase(c.toInt)
 
-  //def isMirrored(c: scala.Char): scala.Boolean
+  def isLowerCase(c: Int): scala.Boolean = {
+    if (c < 256)
+      '\u00AA' == c || '\u00BA' == c || getTypeLT256(c) == LOWERCASE_LETTER
```

Comment:
Prefer `c == '\u00AA'`. In general, we put constants at the right-hand-side of comparisons. (with the exception of the range check, as in `'\u02B0' <= c && c <= '\u02B8'` like you do below)